rotkehlchen/tests/unit/accounting/test_basic.py

```diff
@@ -32,7 +32,7 @@
         {'cost_basis_method': CostBasisMethod.ACB},
         PnlTotals({
             AccountingEventType.TRADE: PNL(taxable=FVal('551.2588760825750541683933333'), free=ZERO),  # noqa: E501
-            AccountingEventType.FEE: PNL(taxable=FVal('-0.2640127597017956302424120028'), free=ZERO),  # noqa: E501
+            AccountingEventType.FEE: PNL(taxable=ZERO, free=ZERO),
```

Comment:
Doesn't the trade one also change when fees are taken into account in cost basis? This is weird and sounds wrong

Reply:
There is no change because of the events that were used for this test.

The event with a fee is #3 in `history1` list. The only event after #3 is event #4.

Then because of:
1. Amount spent by event #4 is quite low.
2. FIFO is used

Only acquisition from event #2 is used and thus the fee change in event #3 is not reflected in the pnl result.